ci-operator/jobs/openshift/osd-utils-cli/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
+- clcollins
+- rogbas
+- fahlmant
+options: {}
+approvers:
```

Comment:
We have `approvers:` twice, shouldn't it reviewers?

Reply:
Updated. Please take a look again